uber/tests/models/test_watchlist.py

```diff
@@ -0,0 +1,56 @@
+from uber.tests import *
+
+
+@pytest.fixture()
+def session(request):
+    session = Session().session
+    request.addfinalizer(session.close)
+    setattr(session, 'watchlist_entry', session.watch_list(first_names='Banned, Alias, Nickname', last_name='Attendee'))
+    return session
+
+
+class TestIfBanned:
+    def test_no_last_name_match(self, session):
+        a = Attendee(first_name='Banned', last_name='Not')
+        session.add(a)
+        assert not a.banned
+
+    def test_only_last_name_match(self, session):
+        a = Attendee(first_name='NotBanned', last_name='Attendee')
+        session.add(a)
+        assert not a.banned
+
+    def test_first_and_last_name_match(self, session):
+        a = Attendee(first_name='Banned', last_name='Attendee')
+        session.add(a)
+        assert a.banned
+
+    def test_email_and_last_name_match(self, session):
+        a = Attendee(email='banned@mailinator.com', last_name='Attendee')
+        session.add(a)
+        assert a.banned
+
+    def test_dob_and_last_name_match(self, session):
+        a = Attendee(last_name='Attendee', birthdate=datetime.strptime('1980-07-10', '%Y-%m-%d').date())
+        session.add(a)
+        assert a.banned
+
+    def test_has_watchlist_entry(self, session):
+        a = Attendee(watchlist_id=session.watchlist_entry.id, first_name='Banned', last_name='Not')
+        session.add(a)
+        assert a.banned
+
+    def test_no_active_entries(self, session):
+        session.watchlist_entry.active = False
+        a = Attendee(first_name='Banned', last_name='Attendee')
+        session.add(a)
+        session.commit()
+        assert not a.banned
+
```

Comment:
Is this behavior what we actually want?  Like, if the same person registers twice and they're on the watchlist, wouldn't we WANT both entries to show up ``banned``?